### PR TITLE
106 add explanatory text to skipped product log message (addendum to PR#109)

### LINKED ISSUE #106: Improve skipped product INFO message
## 💪 Motivation
...so that I can better understand why the product is being skipped.

## 📖 Additional Details

Right now the output message says something like:
```
2022-10-21 08:55:00,875 [INFO] Skipping product /PDS/data/anonymous/PDS/data/PDS4/LADEE/nms_bundle/document/nms_pds_sis.xml
```
which doesn't say anything about why the product was skipped.

## ⚙️ Engineering Details

- [ ] update online registry documentation to describe all the instances when a product is skipped
- [ ] update error message to say explicitly (and succinctly) why the product is being skipped

### PR DESCRIPTION
**Pull request by alexdunnjpl**
## 🗒️ Summary
#109 was incomplete, due to lack of understanding of codebase at the time and failure to comprehensively test change

## ⚙️ Test Data and/or Report
Manually tested.

## ♻️ Related Issues
Fixes #106 
Completes #109

### CODE REVIEW

File: src/main/java/gov/nasa/pds/harvest/dao/MetadataWriter.java [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
 public class MetadataWriter implements Closeable
 {
-    private final static String WARN_SKIP = "Skipping registered product ";
+    private final static String WARN_SKIP_PRE = "Skipping registered product ";
+    private final static String WARN_SKIP_POST = " (LIDVID/LID is is already registered in Elasticsearch)";
```
Comment (nutjob4life) [inline src/main/java/gov/nasa/pds/harvest/dao/MetadataWriter.java:23]: I thought we switched to OpenSearch 🤷

**Review by nutjob4life — APPROVED** [review]
👍

File: src/main/java/gov/nasa/pds/harvest/dao/MetadataWriter.java (hunk 1 above)
Comment (tloubrieu-jpl) [reply src/main/java/gov/nasa/pds/harvest/dao/MetadataWriter.java:23]: Yes we did, let's updat the message to "... already registered in the registry".

**Review by tloubrieu-jpl — CHANGES_REQUESTED** [review]
See Sean's comment

**Comment by alexdunnjpl** [comment]
Oh, good call - I'm so used to autoparsing it that I totally missed it.  Will fix now.

File: src/main/java/gov/nasa/pds/harvest/dao/MetadataWriter.java (hunk 1 above)
Comment (alexdunnjpl) [reply src/main/java/gov/nasa/pds/harvest/dao/MetadataWriter.java]: Fixed

**Comment by tloubrieu-jpl** [comment]
@alexdunnjpl Note also that "is" was duplicated also in the message line 23

**Review by tloubrieu-jpl — APPROVED** [review]
Thanks @alexdunnjpl

**Pull request merged**